### ISSUE TITLE
there is no column view in v0.3.0

### ISSUE THREAD

**Issue by Au-angeloupali** [spacedriveapp/spacedrive#2530]
### Describe the bug

I have been waiting months to finally be able to use the column view in v0.3.0 as mentioned on the roadmap.
Now I have installed it, there is no column view and every time I open the programme it tells me update available even with v0.3.0 installed.

### Platform and versions

```Shell
Windows 11 Pro - 23H2 - 22631.3593
Spacedrive v0.3.0
```

**Comment by zaaakher** [comment]
> every time I open the programme it tells me update available even with v0.3.0 installed.

This part is tracked in #2519

**Comment by Au-angeloupali** [comment]
ok thans, 
and when will the Column View come out instead?

**Comment by ketonik** [comment]
> ok thans, and when will the Column View come out instead?

not until after 'List View' gets disclosure triangles on folders ala macOS Finder ;-)

**Comment by HeavenVolkoff** [comment]
Hey, sorry for ColumnView not being ready yet. It's a common request, and we're aware of it. It's already on our internal TODO list. However, we recently adjusted our priorities a bit, which led to some features being delayed. The roadmap is currently outdated, but we'll update it soon.

**Comment by Au-angeloupali** [comment]
@HeavenVolkoff  When will acrylic mode for windows be activated?

I would like to start using space drive but there are still 2 or 3 elements that make me give up,  SD is a fantastic project. now I use onecommeander (for the column view) but graphically and as a program I realize that spacedrive will be on another level.

**Comment by Au-angeloupali** [comment]
@HeavenVolkoff no way for this?

**Comment by HeavenVolkoff** [comment]
@Au-angeloupali, acrylic mode like this? https://learn.microsoft.com/en-us/windows/apps/design/style/acrylic.

I will need to check if tauri already has support for it, if it does, we can enable it no problem. However, if there is no native support in tauri it would be a low priority feature rn.

About column view, it is high in the list, but there are still a couple of features that need to be worked on before it. When we start implementing it I will send a msg here.

**Comment by onjour** [comment]
> However, we recently adjusted our priorities a bit, which led to some features being delayed. 

May I ask in which way?

**Comment by iLynxcat** [comment]
Keep an eye on #2591 for updates as we've begun to build Columns view!

**Comment by onjour** [comment]
> Hi All, Is there a way to track this feature ?

No clue, releases are stalled anyway so I won't be expecting this anytime soon 💁

**Comment by jamiepine** [comment]
Closing due to v2 rewrite. If you encounter this issue in v2, please open a new issue.
